src/components/Wallet/index.jsx

```diff
@@ -516,6 +520,14 @@ class CreateWallet extends React.Component {
     this.setState({ generating: false });
   };
 
+  handleConfigJsonAddressType = (configJson) => {
+    const config = JSON.parse(configJson);
+    if (config.addressType === "P2WSH-P2SH") {
```

Comment:
Currently we are following coldcard's addressType 'p2wsh_p2sh', but we notice a break change of addressType in coldcard: https://github.com/Coldcard/firmware/blob/65cd37738bd1720ce0abbb4bce5950ce1ce7b007/releases/ChangeLog.md
So I add this func to make it compatible, how do you like it? @bucko13

Reply:
We aims to change the 'p2wsh_p2sh' and 'P2WSH-P2SH' to 'p2sh_p2wsh' and 'P2SH-P2WSH' in Cobo Vault in several days. @humanumbrella

Reply:
Nice. Yes, that makes sense to fix.